solr_configs/orangelight/conf/solrconfig.xml

```diff
@@ -495,9 +487,9 @@
          <str>nameOfCustomComponent2</str>
        </arr>
       -->
-    <arr name="last-components">
+    <!-- <arr name="last-components">
```

Comment:
One more commented block

Reply:
Are you talking only about the 2 lines in 490 or also the rest of the commented blocks from previous commits?